install/sct_changelog.py

```diff
@@ -126,11 +126,19 @@ def check_compatibility(dict_labels=''):
         if items:
             lines.append('\n**{}**\n'.format(label.upper()))
             changelog_pr = changelog_pr.union(set([x['html_url'] for x in items]))
-            items = [" - **%s:** %s. %s[View pull request](%s)" % (",".join(get_sct_function_from_label(x['labels'])),
-                                                                x['title'],
-                                                                check_compatibility(x['labels']),
-                                                                x['html_url']) for x in pulls.get('items')]
-            lines.extend(items)
+            limit = pulls.get('items')
+            for x in pulls.get('items'):
+                if (len(get_sct_function_from_label(x['labels'])) > 0):
+                    items = [" - **%s:** %s. %s[View pull request](%s)" % (
+                    ",".join(get_sct_function_from_label(x['labels'])),
+                    x['title'],
+                    check_compatibility(x['labels']),
+                    x['html_url'])]
+                elif (len(get_sct_function_from_label(x['labels'])) == 0):
```

Comment:
instead of duplicating the code, you can simply do something like (algorithm shown below):
~~~
if number of sct function is more than 0
  items = ": " + list of SCT functions separated by ","
append title
~~~